### PR TITLE
Fix the definition of the remote_collector service

### PR DESCRIPTION
**Pull request by simonpasquier**
This change removes unneeded plugins and adds the ones that are
otherwise required.

### CODE REVIEW

File: heka/meta/heka.yml [hunk 1]
```diff
@@ -97,16 +97,42 @@ metric_collector:
...
+      port: 8326
+      decoder: collectd_decoder
+      splitter: NullSplitter
+  filter:
+    influxdb_accumulator:
+      engine: sandbox
+      module_file: /usr/share/lma_collector/filters/influxdb_accumulator.lua
+      module_dir: /usr/share/lma_collector/common;/usr/share/heka/lua_modules
+      preserve_data: false
+      message_matcher: "Fields[aggregator] == NIL && Type =~ /metric$/"
```
Comment (elemoine) [inline heka/meta/heka.yml]: `Fields[aggregator] == NIL` is unnecessary.

Comment (simonpasquier) [reply heka/meta/heka.yml]: Fixed

File: heka/meta/heka.yml [hunk 2]
```diff
@@ -97,16 +97,42 @@ metric_collector:
...
+      port: 8326
+      decoder: collectd_decoder
+      splitter: NullSplitter
+  filter:
+    influxdb_accumulator:
+      engine: sandbox
+      module_file: /usr/share/lma_collector/filters/influxdb_accumulator.lua
+      module_dir: /usr/share/lma_collector/common;/usr/share/heka/lua_modules
+      preserve_data: false
+      message_matcher: "Type =~ /metric$/"
```
Comment (elemoine) [inline heka/meta/heka.yml:120]: This is minor we could be more explicit:

```
"Type == 'metric' || Type == 'heka.sandbox.metric' || Type == 'heka.sandbox.afd_metric'"
```

Not sure what is best.

**Review by elemoine — APPROVED** [review]
Just added another comment, but this LGTM.

File: heka/meta/heka.yml (hunk 2 above)
Comment (simonpasquier) [reply heka/meta/heka.yml:120]: That's probably for another review. WDYT?

Comment (elemoine) [reply heka/meta/heka.yml:120]: Sure. Just a remark.

**Pull request merged**